$NetBSD: patch-mosquitto.conf,v 1.4 2021/02/01 16:43:54 triaxx Exp $

Log to syslog, instead of (perhaps) not logging.

\todo Figure out what happens if port is not defined, and why we set it.

\todo Think about pidfile; because this can be run in various modes
that seems like it should perhaps be command-line, not config.

--- mosquitto.conf.orig	2021-01-11 16:49:40.000000000 +0000
+++ mosquitto.conf
@@ -148,7 +148,7 @@
 # This should be set to /var/run/mosquitto/mosquitto.pid if mosquitto is
 # being run automatically on boot with an init script and
 # start-stop-daemon or similar.
-#pid_file
+pid_file @VARBASE@/run/mosquitto/mosquitto.pid
 
 # Set to true to queue messages with QoS 0 when a persistent client is
 # disconnected. These messages are included in the limit imposed by
@@ -340,7 +340,7 @@
 # certificate files must have ".crt" as the file ending and you must run
 # "openssl rehash <path to capath>" each time you add/remove a certificate.
 #cafile
-#capath
+capath @SSLCERTS@
 
 
 # If require_certificate is true, you may set use_identity_as_username to true
@@ -448,7 +448,7 @@
 # Note that if the broker is running as a Windows service it will default to
 # "log_dest none" and neither stdout nor stderr logging is available.
 # Use "log_dest none" if you wish to disable logging.
-#log_dest stderr
+log_dest syslog
 
 # Types of messages to log. Use multiple log_type lines for logging
 # multiple types of messages.
